src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1011,11 +1034,73 @@ emit_arm64_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignatur
 				op = SIMD_OP_LLVM_I64ABS;
 				break;
 			}
+		}
 
-			return emit_simd_ins_for_sig (cfg, klass, OP_XOP_X_X, op, arg0_type, fsig, args);
+		case SN_AbsoluteCompareGreaterThan: {
+			return emit_absolute_compare(cfg, klass, fsig, arg0_type, args, SIMD_OP_LLVM_I4FABSOLUTE_COMPARE_GREATER_THAN, SIMD_OP_LLVM_DABSOLUTE_COMPARE_GREATER_THAN);
 		}
+
+	    case SN_AbsoluteCompareGreaterThanOrEqual: {
```

Comment:
This seems like you're mixing spaces and tabs? Indentation looks inconsistent on here.